internal/android_browser.py

```diff
@@ -251,8 +251,8 @@ def on_stop_recording(self, task):
                         logline = '{0:d},{1:d},-1,-1\n'.format(snapshot['time'], snapshot['bw'])
                         logging.debug(logline)
                         gzfile.write(logline)
-                except Exception:
```

Comment:
The `Exception` here is for identifying what kinds of Exceptions the handler wants to catch (all). The [docs](https://docs.python.org/3/tutorial/errors.html) don't really say what the behavior is if it isn't specified.

Reply:
Thanks for clarifying that!